app/channel.go

```diff
@@ -1145,6 +1145,25 @@ func (a *App) GetChannelMembersPage(channelId string, page, perPage int) (*model
 	return result.Data.(*model.ChannelMembers), nil
 }
 
+func (a *App) GetChannelMembersTimezones(channelId string) ([]string, *model.AppError) {
+	var membersTimezones []map[string]string
+	if result := <-a.Srv.Store.Channel().GetChannelMembersTimezones(channelId); result.Err != nil {
```

Comment:
Can you write this adopting the idiomatic error handling we're doing elsewhere?
```go
result := <-a.Srv.Store...
if result.Err != nil {
    // ...
}
membersTimezones = result.Data.([]map[string]string)
```

Reply:
done